lib/web_ui/lib/src/engine/semantics/semantics.dart

```diff
@@ -1439,10 +1465,7 @@ class SemanticsObject {
       recomputePositionAndSize();
     }
 
-    // Ignore pointer events on all container nodes and all platform view nodes.
-    // This is so that the platform views are not obscured by semantic elements
-    // and can be reached by inspecting the web page.
-    if (!hasChildren && !isPlatformView) {
+    if (semanticRole!.acceptsPointerEvents) {
```

Comment:
I'm wondering if we should also check `_rect.isEmpty`. Although, I can't imagine the framework merging a tappable node such that the merged result is empty. So maybe I'm overthinking this.

Reply:
You're worried that we may set `pointer-events: all` on an empty rect? I don't know if this case ever happens or not, but even if it does, I don't see a problem with accepting pointer events on an empty rect that's not going to obscure anything 🙂

Reply:
Fair point!